test/orange/data/testem3.org.omn

```diff
@@ -0,0 +1,238 @@
+! ORANGE testem3 input definition using universes. This geometry supports 
+! simple safety
+[GEOMETRY]
+global "outer"
+
+comp         : matid
+    "Pb"  0
+    "lAr"  1
+
+!#### LEVEL 1 UNIVERSE ####!
+
+[UNIVERSE=general inner]
+interior "pair"
+
+[UNIVERSE][SHAPE=cuboid pair]
+faces 0 0.8 -20 20 -20 20
+
+[UNIVERSE][SHAPE=cuboid gap_shape]
+faces 0 0.23 -20 20 -20 20
+
+[UNIVERSE][SHAPE=cuboid abs_shape]
+faces 0.23 0.8 -20 20 -20 20
+
+[UNIVERSE][CELL gap]
+comp "lAr"
+shapes -gap_shape
+
+[UNIVERSE][CELL absorber]
+comp "Pb"
+shapes -abs_shape
+
+[UNIVERSE=general outer]
+interior "full"
+
+!#### LEVEL 0 UNIVERSE ####!
+
+[UNIVERSE][SHAPE=cuboid full]
```

Comment:
To be consistent with the gdml and "flat" geometry, could you rename this `calorimeter`, and add a cell called `world` that's outside it and inside a box with halfwidth 24?
```py
    {
        '_type': "box",
        'widths': [48, 48, 48],
        'name': "world",
    }
```

Reply:
Sure, noting that the world cell will have an internal region, meaning `supports_safety_` will now be false

Reply:
I don't know if its worth keeping the original geom as well in order to test that this returns true for it:
```
    supports_safety_
        = std::all_of(
              host_data.simple_units[AllItems<SimpleUnitRecord>()].begin(),
              host_data.simple_units[AllItems<SimpleUnitRecord>()].end(),
              [](SimpleUnitRecord const& su) { return su.simple_safety; })
          && host_data.rect_arrays.empty();
          ```

Reply:
Model updated to include world